vector/src/main/res/values/strings.xml

```diff
@@ -3626,4 +3627,18 @@
     <string name="link_this_email_settings_link">Link this email with your account</string>
     <!-- %s will be replaced by the value of link_this_email_settings_link and styled as a link -->
     <string name="link_this_email_with_your_account">%s in Settings to receive invites directly in Element.</string>
+
+    <!-- Poll -->
+    <string name="create_poll_title">Create Poll</string>
+    <string name="create_poll_question_title">Poll question or topic</string>
+    <string name="create_poll_question_hint">Question or topic</string>
+    <string name="create_poll_options_title">Create options</string>
```

Comment:
We discussed about that wording in a Matrix room. Maybe "Options" is better here. What is the status?
CC @gaelledel

Reply:
@gaelledel wants to use "Create options".

Reply:
OK, thanks